src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/GroupedTotalStake/GroupedTotalStakeHeading.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { defineMessages } from 'react-intl';
+
+import { Heading5 } from '~shared/Heading';
+import QuestionMarkTooltip from '~shared/QuestionMarkTooltip';
+
+import styles from './GroupedTotalStakeHeading.css';
+
+const displayName =
+  'common.ColonyActions.ActionDetailsPage.DefaultMotion.StakingWidget.GroupedTotalStakeHeading';
```

Comment:
That is very long. I'm leaning towards making the display name logical instead of a perfect representation of the file structure in the event that would be very long like this one